src/components/SearchableDropdownListItem.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SearchableDropdownListItem = ({ place, handleListItemClick }) => (
+  <a
+    className="dropdown__list__item"
+    onClick={handleListItemClick}
+    onKeyDown={handleListItemClick}
```

Comment:
You can benefit from being in a context of this file and shorten names of your props: 
`handleListItemClick  -> handleClick`

also you may use more generic names like: `handleSelect`